### PR TITLE
Integration of the FAMD by Rudy to the main branch 

### PR DESCRIPTION
**Pull request by rudyusach**
Realisation of a FAMD for data exploration by Rudy

### CODE REVIEW

File: Analyse_Birds.qmd [hunk 1]
```diff
@@ -1,1116 +1,1282 @@
...
+boxplot(diagZ[2,], col='dodgerblue3',ylab='#Rate of nest fate well-classified')
+mean(diagZ[2,])
+```
+
+
+Here we have **58%** of good predictions, which is somewhat below the common threshold of 70% for a model to be considered good. This suggests that the model may need improvement.
+
+This result is not surprising given our initial data. Because they are from different studies some variables barely contains some species or years, etc... This could explain why our model is not suitable. 
+
+## D. FAMD
```
Comment (TanguyAubreton) [inline Analyse_Birds.qmd:457]: les parties de chacun sont avec des grand I, II, III etc

File: Analyse_Birds.qmd [hunk 2]
```diff
@@ -1,1116 +1,1282 @@
...
+
+### Generical exploration
+
+Here, we perform an FAMD on the initial dataframe. The FAMD is used for
+table with qualitative and quanitative variables. The ID are removed
+because of their uselessness.
+
```
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:464]: * quantitative and not quantitative

File: Analyse_Birds.qmd [hunk 3]
```diff
@@ -1,1116 +1,1282 @@
...
+### Conversion of the Year into factor
+data_famd$Year <- as.factor(data_famd$Year)
+
+### Removing of ID
+data_famd <- data_famd |>
+  dplyr::select(-Nest_ID)
+```
+
+We can perform the famd ; it takes some times.
+
```
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:476]: I think you can remove "It takes some time"

File: Analyse_Birds.qmd [hunk 4]
```diff
@@ -1,1116 +1,1282 @@
...
+famd <- FAMD(base = data_famd)
+```
+
+First, let's see the summary :
+
+```{r famd.summary}
+summary(famd)
+```
+
+### Realisation of the FAMD
```
Comment (TanguyAubreton) [inline Analyse_Birds.qmd:487]: realization

File: Analyse_Birds.qmd [hunk 5]
```diff
@@ -1,1116 +1,1282 @@
...
+              )
+
+```
+
+
+### Realisation of a Hierarchical clustering 
+
+We can use the function fviz_cluster to visualise the optimal number of
+clusters to make. It uses the k-means method.
+
```
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:558]: maybe better to say "the k-mean method is used" or "will be used"

File: Analyse_Birds.qmd [hunk 6]
```diff
@@ -1,1116 +1,1282 @@
...
+The green side is characterised by :
+
+-   Year : 2003, 2004, 2005, 2006
+-   County : Dane
+-   State : Wisconsin
+-   Species : Bobolink, Henslow's Sparrow, Eastern Meadowlark
+-   Habitat : Cool-season Grassland
+
+It is necessary to note that several species are located on the counties
```
Comment (VJSTT) [inline Analyse_Birds.qmd:614]: located in and not on

File: Analyse_Birds.qmd [hunk 7]
```diff
@@ -1,1116 +1,1282 @@
...
+             method = "silhouette",
+             FUNcluster = kmeans,
+             linecolor = "green3"
+             )
+```
+
+Three clusters is the best choice according to the k-means method.
+
+We can define classes for a CAH thanks to the function fviz_cluster.
+
```
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:575]: CAH -> in English

File: Analyse_Birds.qmd [hunk 8]
```diff
@@ -1,1116 +1,1282 @@
...
+              repel = FALSE,
+              axes = c(1,2),
+              col.ind = "orange",
+              col.quali.var = "red"
+              )
+
+```
+
+
+### Realisation of a Hierarchical clustering
```
Comment (TanguyAubreton) [inline Analyse_Birds.qmd:554]: pareil : "realization"

File: Analyse_Birds.qmd [hunk 9]
```diff
@@ -1,1116 +1,1282 @@
...
+-   Year : 2003, 2004, 2005, 2006
+-   County : Dane
+-   State : Wisconsin
+-   Species : Bobolink, Henslow's Sparrow, Eastern Meadowlark
+-   Habitat : Cool-season Grassland
+
+It is necessary to note that several species are located on the counties
+near the border between the Illinois and the Wisconsin. Thus they can be
+seen in both states, which can be counter-intuitive by seeing the FAMD.
+However, one factor that can explain this distribution could be years of
```
Comment (VJSTT) [inline Analyse_Birds.qmd:617]: one factor > the only factor

**Review by VJSTT — COMMENTED** [review]
Do the modification please

**Review by OceaneGourdin — COMMENTED** [review]
Why did you change the data file ? (Bird_dataset.csv)

**Review by TanguyAubreton — COMMENTED** [review]
Once you'll had correct the few comments that I put youcan merge with nidification_main !

**Review by TanguyAubreton — APPROVED** [review]

**Pull request merged**